configure.ac

```diff
@@ -40,6 +40,14 @@ AM_INIT_AUTOMAKE([-Wall -Wno-portability foreign])
 
 FLAGS="-Wall -Wextra -mzarch"
 
+dnl --- check for perl-FindBin module
+AC_MSG_CHECKING([if perl module 'FindBin' installed])
```

Comment:
if perl module 'FindBin' **is** installed

Reply:
Done.